resources/views/home.blade.php

```diff
@@ -0,0 +1,23 @@
+@extends('layouts.app')
```

Comment:
chia ra 2 phần : `front_end` and `backend` để tiện xử lý em
resources/views: 
- `front_end`
  + layouts: header, footer, sidebar, master
  + home.blade.php extends master

- `backend`.
  + layouts: header, footer, sidebar, master
  + home.blade.php extends master

Reply:
e tính chia trong resources/views: là admin với client đc ko a